docs/atari/warlords.md

```diff
@@ -37,6 +37,19 @@ When your fortress falls, you receive -1 reward and are done. If you are the las
 
 Environment parameters are common to all Atari environments and are described in the [base Atari documentation](../atari) .
 
+### Action Space (Minimal)
+
+In any given turn, an agent can choose from one of 6 actions.
+
+| Action    | Behavior  |
```

Comment:
Similar to pong, as it uses the same controller https://github.com/Farama-Foundation/Multi-Agent-ALE/blob/master/src/games/supported/Warlords.cpp#L86